### PR TITLE
clean up old files, added Jupyter Notebooks for creating neural network for classifying music genres

### PR DESCRIPTION
**Pull request by christian-eismann**
some code on creating the data set (GTZAN data set), that means splitting the audio files in chunks of 3 seconds, creating spectrograms, creating pytorch dataloader. defining model architecture. 

the model (trained over 500 epochs, 48% accuracy) cannot be provided over github, is too big.

### CODE REVIEW

**Review by vagechirkov — COMMENTED** [review]
Looks good to me!
